build-in-container/run.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+help() {
+    echo "------------ Mariner Build-in-Container ------------"
+    echo "
+    The run.sh script presents these options
+    -t      creates container image
+    -b      creates container, 
+            builds the specs under $(pwd)/SPECS/ 
+            and places the output under $(pwd)/out/
+    -i      create an interactive Mariner build container
+    -c      cleans up the current workspace, container images and instances
+    --help  shows help on usage
+    "
+    echo "----------------------------------------------------"
+}
+
+create_container() {
+    echo "Creating Container Image"
+    source create-build-container.sh
```

Comment:
These paths are not able to find the script if you are using this tool from a repo. 
```bash
cd /path/to/my/CBL-Mariner
/path/to/my/CBL-MarinerTutorials/build-in-container/run.sh -t
```
Gives
```
Creating Container Image
../CBL-MarinerTutorials/build-in-container/run.sh: line 20: create-build-container.sh: No such file or directory
```

Reply:
You should find the script dir based on $0 and save that away. All internal components need to be relative to the script (or just include it all in one file which may be cleaner).